packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -305,6 +305,25 @@ const invokeTask = new tasks.CallApiGatewayRestApiEndpoint(this, 'Call REST API'
 });
 ```
 
+Be aware that the header values must be arrays. When passing the Task Token
```

Comment:
I feel like the native intrinsic function support is a feature in itself and deserves to be mentioned in the readme too.